packages/devtools-reps/src/reps/test-helpers.js

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { shallow } = require("enzyme");
+const { REPS } = require("./rep");
+const { getGripPreviewItems } = require("./rep-utils");
+const { Rep } = REPS;
+const nodeConstants = require("../shared/dom-node-constants");
+
+class TestHelpers {
+  /**
+   * Test that a rep renders correctly across different modes.
+   */
+  static testRepRenderModes(modeTests, testName, componentUnderTest, gripStub,
```

Comment:
I'd prefer to completely remove this helper function.
I find it hard to evolve if needed without breaking other tests, and also, it adds a level of abstraction you have to deal with when writing your tests.
Having directly the code used to render the component directly in the test you're writing makes it simpler to know what's going on.

So, if we remove this function, we might want to architect our tests like https://github.com/devtools-html/devtools-core/pull/436/files#diff-b7f87095025dd0759eb9ce6b643dc158R40 (one `it` block by mode), or like https://github.com/devtools-html/devtools-core/pull/436/files#diff-b7f87095025dd0759eb9ce6b643dc158R40 (one `it` block for all modes).